selfdrive/locationd/locationd.py

```diff
@@ -145,6 +146,32 @@ def handle_log(self, t: float, which: str, msg: capnp._DynamicStructReader) -> H
     elif which == "carState":
       self.car_speed = abs(msg.vEgo)
 
+      if PC:
```

Comment:
@fredyshox can you get something like this merged soon? it's for openpilot on PC without an IMU (doesn't need large scale validation)